site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,141 @@
++++
+date = "2018-07-27T12:21:53-05:00"
+title = "Citra Progress Report - 2018 Q2"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Hello everyone to the summer (or winter, for those on the southern half of this
+Earth) edition of Citra's progess reports. Although not very many changes have
+happened, most have been focused on accuracy improvements and sorely needed user
+experience improvements. Now, enough appetizing, have your entr&eacute;e:
+
+## [Add pause, speed limit hotkeys](https://github.com/citra-emu/citra/pull/3594) by [valentinvanelslande](https://github.com/valentinvanelslande)
+
+Let's start with a simple but sweet one &emdash; you can now assign hotkeys to
+pausing, unpausing, and toggling speed limiting. By default, you can pause with
+<kbd>F4</kbd> and toggle speed limiting with <kbd><kbd>ctrl</kbd>+<kbd>Z</kbd></kbd>.
+But, of course, you can bind these actions to other keys in Citra's `qt-config.ini`.
```

Comment:
Well, you can't slip in a teaser about citra-emu/citra#3786, can you? It's already in Canary, and some users have noticed it. It's perfectly OK if you can't.